lineage/crossval.py

```diff
@@ -1,15 +1,64 @@
 """ Cross validation. """
+import numpy as np
 from scipy.stats import bernoulli
 from copy import deepcopy
+from .Analyze import run_Analyze_over
 
 
 def hide_observation(lineages: list, percentage: float) -> list:
     """ Taking a list of lineages and the percentage of cells want to be masked, it marks those x% negative."""
     new_lineages = deepcopy(lineages)
     for new_lineage in new_lineages:
-        for cell in new_lineage.output_lineage:
-            if bernoulli.rvs(p=percentage, size=1):
-                # negate the cell observations to mask them
-                cell.obs = [-1 * o for o in cell.obs]
+        if len(new_lineage.output_lineage) < 2:
```

Comment:
Why can't you censor a lineage with one cell?

Reply:
my bad, it should and it is fixed.